src/wp-includes/rest-api/class-wp-rest-server.php

```diff
@@ -764,6 +764,7 @@ public static function get_compact_response_links( $response ) {
 	 *
 	 * @param array         $data  Data from the request.
 	 * @param bool|string[] $embed Whether to embed all links or a filtered list of link relations.
+	 *                      Default true.
```

Comment:
Same here.

Reply:
@mukeshpanchal27 Thank you for the review, I've fixed the indention issue and push the changes.